Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -168,9 +182,9 @@ function xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath, b
 %% ------------------------------------------------------------------------------------------------
 %% 3.   Hematocrit & blood T1 correction
 if isfield(x,'Hematocrit')
-        x.Q.BloodT1 = xASL_quant_Hct2BloodT1(x.Hematocrit);
+        x.Q.BloodT1 = xASL_quant_Hct2BloodT1(x.Hematocrit, [], x.MagneticFieldStrength);
```

Comment:
While we're at it, `x.Hematocrit` should probably also go to `x.Q.Hematocrit`. However, it is also a "covariate" (`x.S.SetsName`, `x.S.SetsID`, etc.) from `participants.tsv`

Reply:
We can make that change later in a separate issue if you want. Do you want to create that issue?

Reply:
Yes good idea